test/e2e/radio-group.test.js

```diff
@@ -0,0 +1,24 @@
+const { assert, driver } = require('vl-ui-core').Test.Setup;
+const VlRadioGroupPage = require('./pages/vl-radio-group.page');
+
+describe('vl-radio-group', async () => {
+  const vlRadioGroupPage = new VlRadioGroupPage(driver);
+
+  before(() => {
+    return vlRadioGroupPage.load();
+  });
+
+  it('als gebruiker kan ik maar 1 radio tegelijkertijd aanvinken', async () => {
+    const radio1 = await vlRadioGroupPage.getRadio(1);
+    const radio2 = await vlRadioGroupPage.getRadio(2);
+
+    await assert.eventually.isFalse(radio1.isChecked());
+    await assert.eventually.isFalse(radio2.isChecked());
+    await radio1.click();
```

Comment:
ik zou nog altijd ook nen "check" maken ipv ne click, die maar gewoon doordelegeert naar de click, maar das een betere API vind ik

Reply:
De eindgebruiker klikt, de check API is puur programmatisch. Ik zie geen voordeel.